metadata_backend/api/auth.py

```diff
@@ -66,7 +66,7 @@ async def login(self, req: Request) -> Response:
         # Redirect user to AAI
         response = web.HTTPSeeOther(session["url"])
         response.headers["Location"] = session["url"]
-        raise response
+        return response
```

Comment:
@csc-felipe i think this should have been left as a `raise`, have you tested this with an odic provider ?

this is something we cannot (or it is very difficult) mock with the integration tests, and needs to be tested with a oidc provider

Reply:
The current implementation works with SDS AAI, I have tried it manually, but what kind of tests did you have in mind?

The difference between `raise` and `return` is not so dramatic. afaik, it's more about how the middleware handles them than anything else.

Reply:
https://github.com/CSCfi/metadata-submitter/pull/559